training-maquetado/scss/variables/_sizes.scss

```diff
@@ -1,3 +1,5 @@
-$text-big: 32px;
-$text-medium: 24px;
-$text-small: 20px;
+$text-description-big: 32px;
+$text-description-medium: 24px;
+$text-description-small: 20px;
+$text-list-medium: 16px;
```

Comment:
mmm i wouldn't set the sizes depending on the page. Set xbig etc to the ones that are biger so you can use them in all pages